python/lsst/meas/astrom/matchPessimisticB.py

```diff
@@ -402,8 +402,10 @@ def _doMatch(self, refCat, sourceCat, wcs, refFluxField, numUsableSources,
         # the consensus mode of the matcher. If not default to Optimistic
         # pattern matcher behavior.
         numConsensus = self.config.numPatternConsensus
-        if ref_array.shape[0] < self.config.numBrightStars or \
-           src_array.shape[0] < self.config.numBrightStars:
+        if ref_array.shape[0] < self.config.numBrightStars + \
+           self.config.numPointsForShapeAttempt or \
+           src_array.shape[0] < self.config.numBrightStars + \
+           self.config.numPointsForShapeAttempt:
```

Comment:
Should I make this intermediate variable or would you recommend putting it in the config?